src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -1073,14 +1073,6 @@ private static void AssertTypeFromOperation(SemanticModel model, TypeSymbol expe
             // unlike GetSymbolInfo or GetTypeInfo, GetOperation doesn't use SemanticModel's recovery mode.
             // what that means is that GetOperation might return null for ones GetSymbol/GetTypeInfo do return info from
             // error recovery mode
-            var foreachLoop = decl.Ancestors().OfType<ForEachVariableStatementSyntax>().FirstOrDefault();
```

Comment:
This change causes `GetEnclosingBinderInternalRecovery_11` to validate IOperation nodes now.